kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -16,14 +16,27 @@
     'go_goroutines': 'go_goroutines',
     'APIServiceRegistrationController_depth': 'APIServiceRegistrationController_depth',
     'etcd_object_counts': 'etcd_object_counts',
+
+    # For Kubernetes < 1.14
     'rest_client_request_latency_seconds': 'rest_client_request_latency_seconds',
     'apiserver_admission_webhook_admission_latencies_seconds': 'admission_webhook_admission_latencies_seconds',
     'apiserver_admission_step_admission_latencies_seconds': 'admission_step_admission_latencies_seconds',
-    'apiserver_admission_controller_admission_duration_seconds': 'admission_controller_admission_duration_seconds',
+    'apiserver_admission_controller_admission_latencies_seconds': 'admission_controller_admission_duration_seconds',
     # fmt: off
     'apiserver_admission_step_admission_latencies_seconds_summary':
         'admission_step_admission_latencies_seconds_summary',
     # fmt: on
+
+    # For Kubernetes >= 1.14
+    # (https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.14.md#deprecated-metrics)
+    'rest_client_request_duration_seconds': 'rest_client_request_latency_seconds',
+    'apiserver_admission_webhook_duration_latencies_seconds': 'admission_webhook_admission_latencies_seconds',
+    'apiserver_admission_step_admission_duration_seconds': 'admission_step_admission_latencies_seconds',
+    'apiserver_admission_controller_admission_duration_seconds': 'admission_controller_admission_duration_seconds',
+    # fmt: off
+    'apiserver_admission_step_admission_duration_seconds_summary':
+        'admission_step_admission_latencies_seconds_summary',
```

Comment:
should we transform all these metrics to be `*duration_seconds` instead of `*latencies_seconds`? the downside is loss of continuity if one upgrades their kubernetes version after 1.14, but it may be less confusing in the long run.

Reply:
I also discussed a little bit with @clamoriniere about the opportunity of renaming the Datadog metrics to make them match the new Kube names.
But as you point, renaming a DD metric can cause trouble if there are some monitors using it.
I think it is safer to keep the legacy DD metric name unchanged.